frontend/src/app/components/modals/create-namespace-bucket-modal/create-namespace-bucket-modal.js

```diff
@@ -206,6 +217,13 @@ class CreateNamespaceBucketModalViewModel extends ConnectableViewModel {
             } else if (!writePolicy) {
                 errors.writePolicy = 'Please select a namespace resource';
             }
+        } else if (step === 2) {
+            if (cacheTTL < 0) {
+                errors.cacheTTL = 'Cache TTL must be non-negative integer';
+
+            } else if (cacheTTL > 172800) {
```

Comment:
The max value here is different then the max value as defined in the bucket API schema: 1296000. I am not sure if this is intentional or not.

Reply:
I will figure out and use the same limit.